deploy-if-version-bump.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -e
+if ! git rev-parse `npm run version` 2>/dev/null ; then
+    git tag `npm run version`
+    git push --tags  # update the repository version
```

Comment:
> I have nothing against publishing tags when version updates happen, but this means that we intend to regularly bump the version number (otherwise, this is unnecessary complexity).

> Do we ? The interest of versioning a front in not evident to me, but I'm fine with it if this is an enlighted choice.